packages/assets/scss/03-organisms/_header-alerts.scss

```diff
@@ -3,29 +3,19 @@
 .ma__header-alerts {
   margin-bottom: 30px;
 
-  &__header {
-    cursor: pointer;
-    background-color: $c-warning;
-    padding: 12px 24px;
-    display: flex;
-    align-items: center;
-    flex-wrap: wrap;
-    
-    @media ($bp-x-small-max) {
-      flex-direction: column;
-    }
-    color: $c-black;
-    font-weight: $fonts-light;
-    border: 0;
-    width: 100%;
+  @media ($bp-medium-min) {
```

Comment:
To fix the spacing on the right, I suggest we do it at this level `ma__page-banner__container`

<img width="1132" alt="Screen Shot 2022-04-12 at 1 54 31 PM" src="https://user-images.githubusercontent.com/5789411/163024214-50967960-cdd0-429b-9fe6-6e45e8048a85.png">

Remove the padding left, and `@include ma-container;` for the page banner container

Reply:
The spacing adjustment has been moved to `.mass-alerts-block .ma__page-banner__container`.

Reply:
addressed in https://github.com/massgov/mayflower/pull/1625